### ISSUE TITLE
MD5SUM hits docker pull limit

### ISSUE THREAD

**Issue by berguner** [nf-core/modules#2547]

**Comment by Emiller88** [comment]
@matthdsm Any objections to using https://quay.io/repository/bedrock/ubuntu?

**Comment by matthdsm** [comment]
I don't really care which image you use, just make sure it's not some bloated mess so we can keep the download times low

**Comment by Emiller88** [comment]
With that, we could probably get away alpine for this task, but I'm thinking across the board whether that would work for these minimal containers

**Comment by matthdsm** [comment]
I agree with alpine! We'll have to make sure it uses the same algorithms for everything though, been burned on that before.

**Comment by famosab** [comment]
@edmundmiller @matthdsm spring cleaning 🌻 bump :)

**Comment by vagkaratzas** [comment]
Closing, since docker now pulls from `'community.wave.seqera.io/library/coreutils_grep_gzip_lbzip2_pruned:838ba80435a629f8'`
